app/Models/Foundation/Summit/Factories/SummitOrderFactory.php

```diff
@@ -60,7 +61,15 @@ public static function populate(Summit $summit, SummitOrder $order, array $paylo
             $order->setOwnerEmail(trim($payload['owner_email']));
 
         if (isset($payload['owner_company']) && !is_null($payload['owner_company']))
-            $order->setOwnerCompany(trim($payload['owner_company']));
+            $order->setOwnerCompanyName(trim($payload['owner_company']));
+
+        if (isset($payload['owner_company_id']) && !is_null($payload['owner_company_id'])) {
+            $company = $summit->getRegistrationCompanyById(intval($payload['owner_company_id']));
```

Comment:
please throw a validate exception is the company is not found

Reply:
done